app/models/king.rb

```diff
@@ -29,6 +29,8 @@ def move_too_far?(x, y)
   end
 
   def move_into_position_under_attack?(_x, _y)
-    false # implement
+    # return true if ?--> thinking this would be depending on piece and path
```

Comment:
`move_into_position_under_attack?` At the moment, **all tests pass**. However, there would be different outcomes if king would be moving into position (4,1) where a bishop/queen/rook would be able to attack due to their range if they were in position (0,5) unlike a pawn which is able to move 1 space after initial move-- testing those coordinates showed as valid move when it wouldn't be depending on piece/range -- would that go under implementing in_check?